usage-api/nodejs/index.js

```diff
@@ -1,69 +1,112 @@
-// Variable declarations
-require('dotenv').config();
-let apiKey = process.env.APIKEY;
-let appId = encodeURIComponent(process.env.APPID);
-let request = require('request');
-
-// Call the first function overallCustomerCount()
-overallCustomerCount();
-
-function overallCustomerCount() {
-    let options = {
-        'method': 'GET',
-        'url': 'https://billable-vendor-usage-api.usw2.pure.cloud/v1/regions?appIds=' + appId,
-        'headers': {
+const axios = require('axios');
+
+const apiKey = process.env.APIKEY;
+const appId = encodeURIComponent(process.env.APPID);
+
+/**
+ * Get summary of all regions and the number of orgs
+ * with the premium app.
+ * @returns {Promise}
+ */
+function getRegionSummaries() {
+    return axios.get(`https://billable-vendor-usage-api.usw2.pure.cloud/v1/regions?appIds=${appId}`, {
```

Comment:
Nice example of using an alternative HTTP library.

Reply:
Thanks!